cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsResourceConnector.java

```diff
@@ -235,8 +249,32 @@
 
         AmazonAutoScalingRetryClient amazonASClient = awsClient.createAutoScalingRetryClient(credentialView, regionName);
         saveGeneratedSubnet(ac, stack, cFStackName, cfRetryClient, resourceNotifier);
+
         List<CloudResource> cloudResources = getCloudResources(ac, stack, cFStackName, cfRetryClient, amazonEC2Client, amazonASClient, mapPublicIpOnLaunch);
-        return check(ac, cloudResources);
+
+        cloudResourceStatuses.addAll(check(ac, cloudResources));
+        return cloudResourceStatuses;
+    }
+
+    private List<CloudResourceStatus> buildComputeResourcesForLaunch(AuthenticatedContext ac, CloudStack stack, AdjustmentType adjustmentType, Long threshold) {
```

Comment:
These build* methods call the members of ComputeResourceService, which calls build on all classes that derives from AbstractAwsComputeBuilder. For now, it only has one child, AwsAttachedDiskResourceBuilder, but in the future this can change and the method name you suggest will be misleading.

Reply:
Yes, I agree for the future use. But currently, as our RMP goes, we will only use it for creating disks. I think it would help the reader to reflect this fact somehow. Maybe renaming the build* methods is too much, I agree, but maybe it suffices to rename the local variable to builtVolumes or something like that.